app/src/config.js

```diff
@@ -0,0 +1,17 @@
+const dev = process.env.NODE_ENV === `develop`
```

Comment:
how about just having a `config.json` on the server with specifications for the deployed paths instead of hardcoding all of these values?

Reply:
I'm not 100% I got the server, you mean CircleCi? On S3 we will have only bundled files.

The conversion from json to js was to simplify the process, in this way I bundle and push without need of setup on the destination. Maybe I could use some ENV variables on CircleCi to override localhost, but I would do this anyway with a conf.js file.

Reply:
My idea was to no bundle the `config.js` file but to have a `config.json` that get's loaded from the server when Voyager starts so it is easy to manipulate the properties. Voyager will probably be deployed in different places and different scenarios so hardcoding our values into the config might not be the best idea. But that would need additional work so maybe we leave this as is.

Reply:
Hum sorry I still don't get who is the server. In this moment Voyager will be served as static content (the e2e test could run from circleci artifacts directly once we have a proper network boot system i.e. https://15157-99653950-gh.circle-artifacts.com/0/tmp/voyager/app/dist/index.html#/)

I'll try the ENV variables for RPC / REST and keep only a fallback, sounds good?

Reply:
seems like we can move forward with this for now. but fabo's idea is a good one - so we'll have to find a solution later.

Reply:
I will open an issue explaining my idea. We can move on with this PR without this change. (https://github.com/cosmos/voyager/issues/1951)